extensions/citrus.yaml

```diff
@@ -0,0 +1,9 @@
+---
+group-id: "org.citrusframework"
+artifact-id: "citrus-quarkus"
+versions:
```

Comment:
No need to specify the versions as this will be read from Maven Central in a timely basis

Reply:
ok, so I should only specify the latest version in the PR or completely skip the versions field?

Reply:
You can skip the field completely, but that's fine